app/src/main/java/com/duckduckgo/app/cta/ui/Cta.kt

```diff
@@ -86,11 +86,11 @@ sealed class DaxDialogCta(
     override val appInstallStore: AppInstallStore
 ) : Cta, DialogCta, DaxCta {
 
-    override fun createCta(activity: FragmentActivity): DaxDialog =
+    override fun createCta(context: Context, daxDialogListener: DaxDialogListener): DialogFragment =
         TypewriterDaxDialog.newInstance(
-            daxText = getDaxText(activity),
-            primaryButtonText = activity.resources.getString(okButton),
-            hideButtonText = activity.resources.getString(R.string.daxDialogHideButton),
+            daxText = "",
+            primaryButtonText = "",
```

Comment:
nit: If I recall correctly, these values don't matter because I guess the CTA implementation will take care of doing that.
Do you think this deserves a comment here not to be confused by "this CTA has empty text for primary button or dax text"

Reply:
Done!